openapi/openapi.yaml

```diff
@@ -135,6 +137,15 @@ paths:
           $ref: '#/components/responses/BadRequest'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /city-council-districts:
```

Comment:
Why is this endpoint also being documented in this PR? This should be a part of main so if you rebase off it, it should not show up as a new addition for this PR

Reply:
I moved it from 114 to 149 after capital-projects to alphabetize it

Reply:
Gotcha